packages/uiweb/src/lib/components/space/SpaceWidget/LiveWidgetContent.tsx

```diff
@@ -444,5 +510,47 @@ const RequestsCount = styled.div`
   background-color: ${(props) => props.theme.btnColorPrimary};
   padding: 2px 4px;
   border-radius: 4px;
-  font-size: 12px;
-`;
\ No newline at end of file
+  font-size: 12px;<EmojiContainer />
+`;
+
+const EmojiContainer = styled.div<{ showEmojiContainer: boolean }>`
+  position: relative;
+  bottom: -20px;
+  width: 220px;
+  height: 40px;
+  display: ${(props) => (props.showEmojiContainer ? 'flex' : 'none')};
+  align-items: center;
+  justify-content: center;
+  gap: 12px;
+  border-radius: 99px;
+  border: 1px solid #e5e8f6;
+  background: #fff;
+`;
+
+const SpeakerEmojiContainer = styled.div<{ showEmojiContainer: boolean }>`
+  position: relative;
```

Comment:
general rule of CSS Position: elements nested inside `relative` styled divs should be given position `absolute` so that they can be easily positioned.

Use `position: absolute` to position the elements.